src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -38,6 +39,26 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses {@code attribute string} into an {@code Attribute} and returns it. Leading and trailing whitespaces will
+     * be trimmed.
+     * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
+     */
+    public static Attribute parseAttribute(String attribute) throws ParseException {
+        String trimmedAttribute = attribute.trim();
+        if (trimmedAttribute.equals("-policy")) {
+            return Attribute.POLICY_ID;
+        } else if (trimmedAttribute.equals("-phone")) {
+            return Attribute.PHONE;
+        } else if (trimmedAttribute.equals("-email")) {
+            return Attribute.EMAIL;
+        } else if (trimmedAttribute.equals("-address")) {
+            return Attribute.ADDRESS;
+        } else {
+            throw new ParseException(Attribute.MESSAGE_CONSTRAINTS);
+        }
```

Comment:
One suggestion would be to use switch statements.

Reply:
Done!